image/templates/helm/stackrox-central/values.yaml.htpl

```diff
@@ -193,7 +184,68 @@
 #      # Specify a custom hostname if desired, otherwise accept the default from OpenShift.
 #      host: null
 #
+[< if .FeatureFlags.ROX_POSTGRES_DATASTORE >]
+#  db:
+#    enabled: false
```

Comment:
Does `central.db.enabled` even make sense as a setting, in addition to the feature flag? With the feature flag turned on, will it be at all possible to opt out of postgres?

Reply:
Maybe this is a little future proofing, but in the staged rollout, the feature flag will become an env var (this will also cause a series of problems in this PR with the changes to special casing things with the feature flag) so enabling and disabling will be possible in that context. I can remove in this PR if that's cleaner